### ISSUE TITLE
How to set SecurityLevel for SSH Server

### ISSUE THREAD

**Issue by dandalf** [sshtools/maverick-synergy#11]
How do you set the SecurityLevel to indicate the allowed key ciphers when starting the SshServer? Is there a way to do it after the server is created?

**Comment by ludup** [comment]
It looks like this is not possible to set a preference with the current RC1. Right now, with RC1 you would have to override the createContext method of SshServer, and pass a SecurityLevel in the constructor of the SshServerContext like below:

```
@Override
public SshServerContext createContext(SshEngineContext daemonContext, SocketChannel sc)
		throws IOException, SshException {
	return new SshServerContext(daemonContext.getEngine(), SecurityLevel.PARANOID);
}
```

I've now added a preference field to SshServer for the RC2 builds. This just requires setting the preferred level via:

```
server.setSecurityLevel(SecurityLevel.PARANOID);`
```

That's been committed to the develop branch and should be available in the latest 3.0.0-RC2-SNAPSHOT builds.

**Comment by dandalf** [comment]
Thank you!
